desktop-widgets/mapwidget.cpp

```diff
@@ -6,11 +6,14 @@
 #include "mapwidget.h"

```

Comment:
(I can't comment on the commit message, so I comment here)
That reference of the "previous commit" appears to reference something that isn't there anymore...